Add Quinn K's short-term goal to team goals (#1668)
sourcegraph/about: handbook/product/goals.md

- **Metrics:**
   - Increase visits
   - Increase creation of and usage of extensions
   
### Create baseline for public (and eventually private) code on Sourcegraph Cloud

> - **Owner:** Quinn

Reviewer: ```suggestion
- **Owner:** Quinn K.
```

- **Timeline:** ~2 months
- **Status:** In progress

> - **Outcome:** Any user can add public code from GitHub.com, GitLab.com, and Bitbucket Cloud.[*N<sub>9</sub>*][N9]

Reviewer: The reference numbers are for metrics we don't want to be public (links in a Google Doc). In this case it's just a link to another section of the docs, so we can be clear about what this link is :) 

```suggestion
- **Outcome:** Any user can add public code from GitHub.com, GitLab.com, and Bitbucket Cloud. Support accomplishing [Cloud milestones 1 and 2](https://about.sourcegraph.com/handbook/engineering/cloud#milestones).
```

[N6]: https://docs.google.com/document/d/1l6waCjzVlisYJ1PziNjO9QWF0ExmBMxWg5jWMpt4YW0/edit#bookmark=id.g6oo5rramvpb
[N7]: https://docs.google.com/document/d/1l6waCjzVlisYJ1PziNjO9QWF0ExmBMxWg5jWMpt4YW0/edit#bookmark=id.87kmhwjskbfr
[N8]: https://docs.google.com/document/d/1l6waCjzVlisYJ1PziNjO9QWF0ExmBMxWg5jWMpt4YW0/edit#bookmark=id.cbcufz4r1f61

> [N9]: https://about.sourcegraph.com/handbook/engineering/cloud#milestones

Reviewer: ```suggestion
```
